src/styles/book-item.css

```diff
@@ -0,0 +1,187 @@
+.item-panel {
+  width: 75rem;
+  height: min-content;
+  margin: 2.313rem 6.25rem 0;
+  padding: 2rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.item-category {
+  margin: 0 2.438rem 0 0;
+  opacity: 0.5;
+  font-family: Montserrat, serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.item-title {
+  margin: 0.188rem 0;
+  font-family: RobotoSlab, serif;
```

Comment:
- [ ] It would be better if we implement the DRY principle in this case; once we have repeating rules, there are many ways for example creating a class for it like `.fav_primary_font { font-family: RobotoSlab, serif; }` or you can group the classes with the repeated rules for example; `.class1, .class2 { font-family: RobotoSlab, serif; }`